### PR TITLE
Remove load_pkcs12() function, kubernetes code, and PyOpenSSL

### PR DESCRIPTION
**Pull request by btam**
Fix vulnerabilities in https://lists.debian.org/debian-security-announce/2024/msg00018.html and https://github.com/advisories/GHSA-3ww4-gg4f-jr7f

Cryptography 42.0.3 removed the `crypto.load_pkcs12()` function, so we had to work around that.

Removed the kubernetes secret handling code and the Dockerfile because we don't use them in our image.

Thanks to Ang Li for this PR that I based my changes off of: https://msazure.visualstudio.com/Cognitive%20Services/_git/Hosting-Tools/pullrequest/9558108

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -31,12 +31,12 @@
 import base64
 import requests
 
+from cryptography.hazmat.primitives import serialization
```
Comment (Michael-Sinz) [inline app/main.py:34]: For now this looks good.  The comments from the developers of the hazmat are:

image

**Review by Michael-Sinz — APPROVED** [review]
Funny how we need to use the parts of cryptography that is "hazardous" :-)

File: app/main.py (hunk 1 above)
Comment (Michael-Sinz) [reply app/main.py:34]: Just glad it does not have sharks with lasers :-)

Comment (btam) [reply app/main.py:34]: Yeah, I hesitated because I've never fought a dinosaur with a laser gun before.

**Pull request merged**